lib/main_common.pm

```diff
@@ -467,6 +469,8 @@ sub load_zdup_tests {
 sub load_autoyast_tests {
     #    init boot in load_boot_tests
     loadtest("autoyast/installation");
+    #   library function like send_key or reboot will not work, therefore exiting earlier
+    return loadtest "locale/keymap_or_locale" if get_var('INSTALL_KEYBOARD_LAYOUT');
```

Comment:
I know that this variable was there before, but could you please so kind and add description to `variables.md`?